src/rospkg/rospack.py

```diff
@@ -157,14 +160,18 @@ def get_ros_paths(self):
         return self._ros_paths[:]
     ros_paths = property(get_ros_paths, doc="Get ROS paths of this instance")
 
-    def get_manifest(self, name):
+    def get_manifest(self, name, typeof_depend=0):
         """
+        :param typeof_depend:
+                0: all kinds of depend.
+                1: only runtime_depend, such as run_depend in package.xml format 1 and 2.
+                2: only build_depend.
         :raises: :exc:`InvalidManifest`
         """
         if name in self._manifests:
             return self._manifests[name]
         else:
-            return self._load_manifest(name)
+            return self._load_manifest(name, typeof_depend)
```

Comment:
Please call keyword arguments explicitly using the keyword rather than as a positional argument.

Same below.